app/src/main/java/org/simple/clinic/patient/PatientRepository.kt

```diff
@@ -12,6 +12,7 @@ import org.simple.clinic.facility.Facility
 import org.simple.clinic.overdue.Appointment.AppointmentType.Manual
```

Comment:
Can you please add/update tests for repositories whenever you make changes to the data layer to ensure that they are working as expected?